node-require/divide.js

```diff
@@ -0,0 +1,5 @@
+const divide = (a, b) => {
```

Comment:
Since you are using an arrow function, you could simplify it by removing {}:
const divide = (a, b) => a / b;